### ISSUE TITLE
Fields Aggregation with Elastalert

### ISSUE THREAD

**Issue by vikasgurlinka** [Yelp/elastalert#2944]
Hi @Qmando @danielpops @snuids @JeffAshton @alvarolmedo ,

Is there any way to sum up two values and trigger an alert with elastalert.
I have two fields for disk utilization. I would like to sum up those two fields and trigger an alert if it breaches the threshold.

if disk_1+disk_2 > threshold  then trigger an alert

--
Thanks

**Comment by Qmando** [comment]
I don't think this is currently possible out of the box. You'd probably have to write a custom rule type.
